samples/highcharts/demo/combo-data-resolution/demo.js

```diff
@@ -0,0 +1,47 @@
+Highcharts.chart('container', {
+    title: {
+        text: 'Global temperature mean anomalies from 1880 to 2016',
+        align: 'left'
+    },
+    accessibility: {
+        description: `
+            A chart of monthly mean temperature anomalies from 1880 to 2016.
+            It employs two series, one column and one line,
+            to convey the same data at different resolutions.
+            There is only one column per thirty years,
+            but the line series have a datapoint per year.
+        `
+    },
+    series: [{
+        type: 'column',
+        name: 'Average per thirty years',
+        data: JSON.parse(document.getElementById('columnData').textContent)
+    }, {
+        name: 'Monthly average',
+        data: JSON.parse(document.getElementById('lineData').textContent),
+        description: `
+            A data series illustrating the same data as the first,
+            but at different time intervals
+        `
+    }],
+    tooltip: {
```

Comment:
The tooltip for the monthly average shows dates like `31 May, 2019`. Check the timezone. Make sure that the values are actually set on midnight on the first of each month. The resulting tooltip should show something like 'May 2019` with no date or hour specified.